x-pack/legacy/plugins/apm/public/components/shared/MetadataTable/__test__/Section.test.tsx

```diff
@@ -11,7 +11,9 @@ import { expectTextsInDocument } from '../../../../utils/testHelpers';
 
 describe('Section', () => {
   it('shows "empty state message" if no data is available', () => {
-    const output = render(<Section />);
+    let output = render(<Section />);
+    expectTextsInDocument(output, ['No data available']);
+    output = render(<Section keyValuePairs={[]} />);
```

Comment:
Instead of replacing `output` can you use two different `const`? `let` statements always forces me to read things a bit closer to figure out what mutation is happening.